dev/com.ibm.websphere.appserver.features/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
+auto.sync=false
+build.scans.enabled=false
+connection.gradle.distribution=GRADLE_DISTRIBUTION(VERSION(6.3))
```

Comment:
I don't think you want to check this file in.

Reply:
@jhanders34 do you think this eclipse file should be in the .gitignore?

Reply:
In general we do not use the buildship eclipse nature.  You will not have a good experience using the buildship gradle nature in eclipse for OpenLiberty.  It is best to import your projects as existing projects and not as gradle projects using the buildship plugin.  Were you seeing some benefits using the gradle buildship plugin?

Reply:
Thanks tevans78 and jhanders24. I'm Volod will be working with me today to see where I went wrong and review the delivery process.